runtime/pkg/rilltime/rilltime.go

```diff
@@ -96,75 +106,94 @@ var (
 		timeutil.TimeGrainHour:    timeutil.TimeGrainMinute,
 		timeutil.TimeGrainDay:     timeutil.TimeGrainHour,
 		timeutil.TimeGrainWeek:    timeutil.TimeGrainDay,
-		timeutil.TimeGrainMonth:   timeutil.TimeGrainWeek,
+		timeutil.TimeGrainMonth:   timeutil.TimeGrainDay,
 		timeutil.TimeGrainQuarter: timeutil.TimeGrainMonth,
 		timeutil.TimeGrainYear:    timeutil.TimeGrainMonth,
 	}
 )
 
 type Expression struct {
-	From           *Link          `parser:"@@"`
-	To             *Link          `parser:"(To @@)?"`
-	Grain          *string        `parser:"(By @Grain)?"`
-	AnchorOverride *LabeledAnchor `parser:"('@' @@)?"`
-	TimeZone       *string        `parser:"('@' @TimeZone)?"`
+	Interval        *Interval      `parser:"@@"`
+	AnchorOverrides []*PointInTime `parser:"(As Of @@)*"`
+	Grain           *string        `parser:"(By @Grain)?"`
+	TimeZone        *string        `parser:"('@' @TimeZone)?"`
 
 	isNewFormat bool
 	timeZone    *time.Location
```

Comment:
Expression has two timeZone fields ? Shall we merge them into one or rename to make it clear which one to use when ?